diff --git a/src/Entity/MediaBundle.php b/src/Entity/MediaBundle.php
index 8833f82..df51c81 100644
--- a/src/Entity/MediaBundle.php
+++ b/src/Entity/MediaBundle.php
@@ -16,6 +16,7 @@ use Drupal\media_entity\MediaInterface;
  *   id = "media_bundle",
  *   label = @Translation("Media bundle"),
  *   handlers = {
+ *     "access" = "Drupal\media_entity\MediaBundleAccessControlHandler",
  *     "form" = {
  *       "add" = "Drupal\media_entity\MediaBundleForm",
  *       "edit" = "Drupal\media_entity\MediaBundleForm",
diff --git a/src/MediaBundleAccessControlHandler.php b/src/MediaBundleAccessControlHandler.php
new file mode 100644
index 0000000..847d685
--- /dev/null
+++ b/src/MediaBundleAccessControlHandler.php
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\media_entity;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityAccessControlHandler;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Defines the access control handler for the media bundle entity type.
+ *
+ * @see \Drupal\media_entity\Entity\MediaBundle
+ */
+class MediaBundleAccessControlHandler extends EntityAccessControlHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function checkAccess(EntityInterface $entity, $operation, AccountInterface $account) {
+    switch ($operation) {
+      case 'view':
+        return AccessResult::allowedIfHasPermission($account, 'access content');
+    }
+  }
+
+}
